app/modelmodule/admin_acls.py

```diff
@@ -0,0 +1,65 @@
+"""Model for admin/moderator access control."""
+
+
+from google.appengine.ext import db
+
+
+class AdminPermission(db.Model):
+    """Entity class for tracking admin/moderator access.
+
+    There should be only one AdminPermission entity for any given repo/email
```

Comment:
can that constraint be enforced by a 
class AdminPermission(db.Model):
  class Meta(object):
    unique_together = (('repo', 'email_address'),)

Reply:
Not exactly, because `unique_together` is part of the Django ORM, which we're not using. However, the key is defined by these two fields, and the key has to be unique, so it kinda works out the same.